src/s2wasm.h

```diff
@@ -1342,6 +1346,40 @@ class S2WasmBuilder {
     }
   }
 
+  // Converts invoke wrapper names generated by LLVM backend to real invoke
```

Comment:
Pretty much everything in here is LLVM wasm-backend specific right now. It's possible that we generalize and document the format, but we're not so far along even defining what we want that I think we should split these hairs. I would argue that the fact that we have emscripten-specific exception behavior in here is just as weird if not more so. Anyway,  maybe `fixEmExceptionInvoke` or some such would be a more-descriptive name?